### ISSUE TITLE
2025-04-09, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#135]
# Today's Arxiv Papers

(1) [GOLLuM: Gaussian Process Optimized LLMs -- Reframing LLM Finetuning through Bayesian Optimization](http://arxiv.org/abs/2504.06265v1), 2025-04-08

(2) [Hogwild! Inference: Parallel LLM Generation via Concurrent Attention](http://arxiv.org/abs/2504.06261v1), 2025-04-08

(3) [FEABench: Evaluating Language Models on Multiphysics Reasoning Ability](http://arxiv.org/abs/2504.06260v1), 2025-04-08

(4) [Transfer between Modalities with MetaQueries](http://arxiv.org/abs/2504.06256v1), 2025-04-08

(5) [LExT: Towards Evaluating Trustworthiness of Natural Language Explanations](http://arxiv.org/abs/2504.06227v1), 2025-04-08

(6) [Encoder-Decoder Gemma: Improving the Quality-Efficiency Trade-Off via Adaptation](http://arxiv.org/abs/2504.06225v1), 2025-04-08

(7) [Can Performant LLMs Be Ethical? Quantifying the Impact of Web Crawling Opt-Outs](http://arxiv.org/abs/2504.06219v1), 2025-04-08

(8) [From 128K to 4M: Efficient Training of Ultra-Long Context Large Language Models](http://arxiv.org/abs/2504.06214v1), 2025-04-08

(9) [TxGemma: Efficient and Agentic LLMs for Therapeutics](http://arxiv.org/abs/2504.06196v1), 2025-04-08

(10) [Assessing how hyperparameters impact Large Language Models' sarcasm detection performance](http://arxiv.org/abs/2504.06166v1), 2025-04-08

(11) [Navigating the Rabbit Hole: Emergent Biases in LLM-Generated Attack Narratives Targeting Mental Health Groups](http://arxiv.org/abs/2504.06160v1), 2025-04-08

(12) [V-MAGE: A Game Evaluation Framework for Assessing Visual-Centric Capabilities in Multimodal Large Language Models](http://arxiv.org/abs/2504.06148v1), 2025-04-08

(13) [ARLO: A Tailorable Approach for Transforming Natural Language Software Requirements into Architecture using LLMs](http://arxiv.org/abs/2504.06143v1), 2025-04-08

(14) [QGen Studio: An Adaptive Question-Answer Generation, Training and Evaluation Platform](http://arxiv.org/abs/2504.06136v1), 2025-04-08

(15) [Knowledge Graph Completion with Relation-Aware Anchor Enhancement](http://arxiv.org/abs/2504.06129v1), 2025-04-08

(16) [Leanabell-Prover: Posttraining Scaling in Formal Reasoning](http://arxiv.org/abs/2504.06122v1), 2025-04-08

(17) [Nonuniform-Tensor-Parallelism: Mitigating GPU failure impact for Scaled-up LLM Training](http://arxiv.org/abs/2504.06095v1), 2025-04-08

(18) [Multi-Sense Embeddings for Language Models and Knowledge Distillation](http://arxiv.org/abs/2504.06036v1), 2025-04-08

(19) [CAI: An Open, Bug Bounty-Ready Cybersecurity AI](http://arxiv.org/abs/2504.06017v1), 2025-04-08

(20) [Llama-3-Nanda-10B-Chat: An Open Generative Large Language Model for Hindi](http://arxiv.org/abs/2504.06011v1), 2025-04-08

(21) [Optuna vs Code Llama: Are LLMs a New Paradigm for Hyperparameter Tuning?](http://arxiv.org/abs/2504.06006v1), 2025-04-08

(22) [NativQA Framework: Enabling LLMs with Native, Local, and Everyday Knowledge](http://arxiv.org/abs/2504.05995v1), 2025-04-08

(23) [Unsupervised Location Mapping for Narrative Corpora](http://arxiv.org/abs/2504.05954v1), 2025-04-08

(24) [InstructMPC: A Human-LLM-in-the-Loop Framework for Context-Aware Control](http://arxiv.org/abs/2504.05946v1), 2025-04-08

(25) [Assessing Thai Dialect Performance in LLMs with Automatic Benchmarks and Human Evaluation](http://arxiv.org/abs/2504.05898v1), 2025-04-08

(26) [HybriMoE: Hybrid CPU-GPU Scheduling and Cache Management for Efficient MoE Inference](http://arxiv.org/abs/2504.05897v1), 2025-04-08

(27) [Agent Guide: A Simple Agent Behavioral Watermarking Framework](http://arxiv.org/abs/2504.05871v1), 2025-04-08

(28) [CTI-HAL: A Human-Annotated Dataset for Cyber Threat Intelligence Analysis](http://arxiv.org/abs/2504.05866v1), 2025-04-08

(29) [Are Generative AI Agents Effective Personalized Financial Advisors?](http://arxiv.org/abs/2504.05862v1), 2025-04-08

(30) [Enhancing Coreference Resolution with Pretrained Language Models: Bridging the Gap Between Syntax and Semantics](http://arxiv.org/abs/2504.05855v1), 2025-04-08

(31) [PathGPT: Leveraging Large Language Models for Personalized Route Generation](http://arxiv.org/abs/2504.05846v1), 2025-04-08

(32) [Leveraging Robust Optimization for LLM Alignment under Distribution Shifts](http://arxiv.org/abs/2504.05831v1), 2025-04-08

(33) [Right Question is Already Half the Answer: Fully Unsupervised LLM Reasoning Incentivization](http://arxiv.org/abs/2504.05812v1), 2025-04-08

(34) [PaMi-VDPO: Mitigating Video Hallucinations by Prompt-Aware Multi-Instance Video Preference Learning](http://arxiv.org/abs/2504.05810v1), 2025-04-08

(35) [StealthRank: LLM Ranking Manipulation via Stealthy Prompt Optimization](http://arxiv.org/abs/2504.05804v1), 2025-04-08

(36) [From Superficial to Deep: Integrating External Knowledge for Follow-up Question Generation Using Knowledge Graph and LLM](http://arxiv.org/abs/2504.05801v1), 2025-04-08

(37) [How to Enable LLM with 3D Capacity? A Survey of Spatial Reasoning in LLM](http://arxiv.org/abs/2504.05786v1), 2025-04-08

(38) [MDK12-Bench: A Multi-Discipline Benchmark for Evaluating Reasoning in Multimodal Large Language Models](http://arxiv.org/abs/2504.05782v1), 2025-04-08

(39) [Layer-Aware Embedding Fusion for LLMs in Text Classifications](http://arxiv.org/abs/2504.05764v1), 2025-04-08

(40) [SEA-LION: Southeast Asian Languages in One Network](http://arxiv.org/abs/2504.05747v1), 2025-04-08

(41) [LLM-assisted Mutation for Whitebox API Testing](http://arxiv.org/abs/2504.05738v1), 2025-04-08

(42) [Rank-Then-Score: Enhancing Large Language Models for Automated Essay Scoring](http://arxiv.org/abs/2504.05736v1), 2025-04-08

(43) [LLM$\times$MapReduce-V2: Entropy-Driven Convolutional Test-Time Scaling for Generating Long-Form Articles from Extremely Long Resources](http://arxiv.org/abs/2504.05732v1), 2025-04-08

(44) [Retrieval Augmented Generation with Collaborative Filtering for Personalized Text Generation](http://arxiv.org/abs/2504.05731v1), 2025-04-08

(45) [Unified Generative Search and Recommendation](http://arxiv.org/abs/2504.05730v1), 2025-04-08

(46) [Single-Agent vs. Multi-Agent LLM Strategies for Automated Student Reflection Assessment](http://arxiv.org/abs/2504.05716v1), 2025-04-08

(47) [Automated Archival Descriptions with Federated Intelligence of LLMs](http://arxiv.org/abs/2504.05711v1), 2025-04-08

(48) [Large Language Models Enhanced Hyperbolic Space Recommender Systems](http://arxiv.org/abs/2504.05694v1), 2025-04-08

(49) [STRIVE: A Think & Improve Approach with Iterative Refinement for Enhancing Question Quality Estimation](http://arxiv.org/abs/2504.05693v1), 2025-04-08

(50) [StayLTC: A Cost-Effective Multimodal Framework for Hospital Length of Stay Forecasting](http://arxiv.org/abs/2504.05691v1), 2025-04-08

(51) [Separator Injection Attack: Uncovering Dialogue Biases in Large Language Models Caused by Role Separators](http://arxiv.org/abs/2504.05689v1), 2025-04-08

(52) [Towards Smarter Hiring: Are Zero-Shot and Few-Shot Pre-trained LLMs Ready for HR Spoken Interview Transcript Analysis?](http://arxiv.org/abs/2504.05683v1), 2025-04-08

(53) [On the Suitability of Reinforcement Fine-Tuning to Visual Tasks](http://arxiv.org/abs/2504.05682v1), 2025-04-08

(54) [VC-LLM: Automated Advertisement Video Creation from Raw Footage using Multi-modal LLMs](http://arxiv.org/abs/2504.05673v1), 2025-04-08

(55) [Sugar-Coated Poison: Benign Generation Unlocks LLM Jailbreaking](http://arxiv.org/abs/2504.05652v1), 2025-04-08

(56) [Leveraging Prompt-Tuning for Bengali Grammatical Error Explanation Using Large Language Models](http://arxiv.org/abs/2504.05642v1), 2025-04-08

(57) [TAGC: Optimizing Gradient Communication in Distributed Transformer Training](http://dx.doi.org/10.1145/3721146.3721946), 2025-04-08

(58) [Reasoning Towards Fairness: Mitigating Bias in Language Models through Reasoning-Guided Fine-Tuning](http://arxiv.org/abs/2504.05632v1), 2025-04-08

(59) [Model-Agnostic Policy Explanations with Large Language Models](http://arxiv.org/abs/2504.05625v1), 2025-04-08

(60) [Two Intermediate Translations Are Better Than One: Fine-tuning LLMs for Document-level Translation Refinement](http://arxiv.org/abs/2504.05614v1), 2025-04-08

(61) [FactGuard: Leveraging Multi-Agent Systems to Generate Answerable and Unanswerable Questions for Enhanced Long-Context LLM Extraction](http://arxiv.org/abs/2504.05607v1), 2025-04-08

(62) [ShadowCoT: Cognitive Hijacking for Stealthy Reasoning Backdoors in LLMs](http://arxiv.org/abs/2504.05605v1), 2025-04-08

(63) [On the Impact of Language Nuances on Sentiment Analysis with Large Language Models: Paraphrasing, Sarcasm, and Emojis](http://arxiv.org/abs/2504.05603v1), 2025-04-08

(64) [Skywork R1V: Pioneering Multimodal Reasoning with Chain-of-Thought](http://arxiv.org/abs/2504.05599v1), 2025-04-08

(65) [DEL: Context-Aware Dynamic Exit Layer for Efficient Self-Speculative Decoding](http://arxiv.org/abs/2504.05598v1), 2025-04-08

(66) [Knowledge-Instruct: Effective Continual Pre-training from Limited Data using Instructions](http://arxiv.org/abs/2504.05571v1), 2025-04-08

(67) [Can Large Language Models Match Tutoring System Adaptivity? A Benchmarking Study](http://arxiv.org/abs/2504.05570v1), 2025-04-07

(68) [From Fairness to Truthfulness: Rethinking Data Valuation Design](http://arxiv.org/abs/2504.05563v1), 2025-04-07

(69) [SciSciGPT: Advancing Human-AI Collaboration in the Science of Science](http://arxiv.org/abs/2504.05559v1), 2025-04-07

(70) [COIG-P: A High-Quality and Large-Scale Chinese Preference Dataset for Alignment with Human Values](http://arxiv.org/abs/2504.05535v1), 2025-04-07

(71) [Bridging Industrial Expertise and XR with LLM-Powered Conversational Agents](http://arxiv.org/abs/2504.05527v1), 2025-04-07

(72) [Pretraining Language Models for Diachronic Linguistic Change Discovery](http://arxiv.org/abs/2504.05523v1), 2025-04-07

(73) [User Feedback Alignment for LLM-powered Exploration in Large-scale Recommendation Systems](http://arxiv.org/abs/2504.05522v1), 2025-04-07

(74) [Efficient Reinforcement Finetuning via Adaptive Curriculum Learning](http://arxiv.org/abs/2504.05520v1), 2025-04-07

(75) [Evaluating the Generalization Capabilities of Large Language Models on Code Reasoning](http://arxiv.org/abs/2504.05518v1), 2025-04-07

(76) [Prism: Dynamic and Flexible Benchmarking of LLMs Code Generation with Monte Carlo Tree Search](http://arxiv.org/abs/2504.05500v1), 2025-04-07

(77) [A Survey on Hypothesis Generation for Scientific Discovery in the Era of Large Language Models](http://arxiv.org/abs/2504.05496v1), 2025-04-07

(78) [REEF: Relevance-Aware and Efficient LLM Adapter for Video Understanding](http://arxiv.org/abs/2504.05491v1), 2025-04-07

(79) [Imperative vs. Declarative Programming Paradigms for Open-Universe Scene Generation](http://arxiv.org/abs/2504.05482v1), 2025-04-07

(80) [GraphRAFT: Retrieval Augmented Fine-Tuning for Knowledge Graphs on Graph Databases](http://arxiv.org/abs/2504.05478v1), 2025-04-07

(81) [Less but Better: Parameter-Efficient Fine-Tuning of Large Language Models for Personality Detection](http://arxiv.org/abs/2504.05411v1), 2025-04-07

(82) [URECA: Unique Region Caption Anything](http://arxiv.org/abs/2504.05305v1), 2025-04-07

(83) [Truthful or Fabricated? Using Causal Attribution to Mitigate Reward Hacking in Explanations](http://arxiv.org/abs/2504.05294v1), 2025-04-07

(84) [EduPlanner: LLM-Based Multi-Agent Systems for Customized and Intelligent Instructional Design](http://arxiv.org/abs/2504.05370v1), 2025-04-07

(85) [The challenge of uncertainty quantification of large language models in medicine](http://arxiv.org/abs/2504.05278v1), 2025-04-07

(86) [Enhancing LLM-Based Short Answer Grading with Retrieval-Augmented Generation](http://arxiv.org/abs/2504.05276v1), 2025-04-07

(87) [Do PhD-level LLMs Truly Grasp Elementary Addition? Probing Rule Learning vs. Memorization in Large Language Models](http://arxiv.org/abs/2504.05262v1), 2025-04-07

(88) [How to evaluate control measures for LLM agents? A trajectory from today to superintelligence](http://arxiv.org/abs/2504.05259v1), 2025-04-07

(89) [Learning to Reason Over Time: Timeline Self-Reflection for Improved Temporal Reasoning in Language Models](http://arxiv.org/abs/2504.05258v1), 2025-04-07

(90) [Explaining Low Perception Model Competency with High-Competency Counterfactuals](http://arxiv.org/abs/2504.05254v1), 2025-04-07

(91) [LLM-based Automated Grading with Human-in-the-Loop](http://arxiv.org/abs/2504.05239v1), 2025-04-07

(92) [Mapping biodiversity at very-high resolution in Europe](http://arxiv.org/abs/2504.05231v1), 2025-04-07

(93) [Leveraging LLMs for Utility-Focused Annotation: Reducing Manual Effort for Retrieval and RAG](http://arxiv.org/abs/2504.05220v2), 2025-04-07

(94) [LLM-Alignment Live-Streaming Recommendation](http://arxiv.org/abs/2504.05217v1), 2025-04-07

(95) [Unleashing the Power of LLMs in Dense Retrieval with Query Likelihood Modeling](http://arxiv.org/abs/2504.05216v1), 2025-04-07

(96) [Post-Training Language Models for Continual Relation Extraction](http://arxiv.org/abs/2504.05214v1), 2025-04-07

(97) [Quantum Program Linting with LLMs: Emerging Results from a Comparative Study](http://arxiv.org/abs/2504.05204v1), 2025-04-07

(98) [Concise Reasoning via Reinforcement Learning](http://arxiv.org/abs/2504.05185v1), 2025-04-07

(99) [BRIDGES: Bridging Graph Modality and Large Language Models within EDA Tasks](http://arxiv.org/abs/2504.05180v1), 2025-04-07

(100) [Evaluating Knowledge Graph Based Retrieval Augmented Generation Methods under Knowledge Incompleteness](http://arxiv.org/abs/2504.05163v1), 2025-04-07
